### DISCUSSION TITLE
light exposure question

### DISCUSSION THREAD

**Discussion by 11jing** [activityMonitoring/biobankAccelerometerAnalysis#257] [Q&A] [answered]
hello, i am an epidemiology phd student  at vienna, i am also using the ukbiobank accelerometer data for the health information extraction. i am curious about the light eposure information extraction, and i want to ask if i want to extract, which package or which document i should start exploring? thanks very much! looking forward to hearing from you!

**Comment by angerhang** [comment, marked as answer]
https://github.com/OxWearables/actipy provides light extraction

**Reply by 11jing** [reply]
thanks for your answer, i will try this method later in my computer. but i check the code  and output component inside, i didn't see the information related to the light exposure, could you tell me a little bit detailed info

**Reply by chanshing** [reply]
Apologies @11jing you're right
![image](https://github.com/user-attachments/assets/1583c4ec-fa59-4ba1-8d8a-28c3c613133a)

Documentation is outdated and we need to update it, but if you run it you should be able to see a fifth column "light".

**Comment by 11jing** [comment]
thanks for your reply! i will run it and see then!
